INSTALLATION.md

```diff
@@ -13,44 +15,54 @@ brew tap stackitcloud/tap
 2. You can then install the CLI via:
 
 ```shell
-brew install stackit-cli
+brew install stackit
 ```
 
-## Linux
+### Linux
 
-We will soon distribute the STACKIT CLI via [Snap](https://snapcraft.io/). For the moment, you can either install via [Homebrew](https://brew.sh/) or refer to the [manual installation](#manual-installation) guide.
+We are working on distribution the CLI using a package manager for Linux. For the moment, you can either install via [Homebrew](https://brew.sh/) or refer to the [manual installation](#manual-installation) guide.
 
-## Windows
+### Windows
 
-We will soon distribute the STACKIT CLI via [Chocolatey](https://chocolatey.org/). For the moment, please refer to the [manual installation](#manual-installation) guide.
+We are working on distribution the CLI using a package manager for Windows. For the moment, please refer to the [manual installation](#manual-installation) guide.
 
 ## Manual installation
 
-Alternatively, you can get the STACKIT CLI by downloading a pre-compiled binary or compiling it from source.
+You can also get the STACKIT CLI by compiling it from source or downloading a pre-compiled binary.
 
-### Pre-compiled binary
+### Compile from source
 
-1. Download the binary corresponding to your operating system and CPU architecture from our [Releases](https://github.com/stackitcloud/stackit-cli/releases) page
-2. Extract the contents of the file to your file system and move it to your preferred location (e.g. your home directory)
-3. (For macOS only) Right click on the executable, select "Open". You will see a dialog stating the identity of the developer cannot be confirmed. Click on "Open" to allow the app to run on your Mac. We soon plan to certificate the STACKIT CLI to be trusted by macOS
+If you have Go 1.16+ installed, you can directly install via:
 
-### Compile from source
+```shell
+go install github.com/stackitcloud/stackit-cli@latest
+```
+
+> For more information, please refer to the [`go install` documentation](https://go.dev/ref/mod#go-install)
+
+Alternativelly, you can:
 
 1. Clone the repository
 2. Build the application locally by running:
 
    ```bash
-   $ make build
+   make build

```

Comment:
We don't need the `$` as we are already inside a shell code block and the user can just copy the command using 
![image](https://github.com/stackitcloud/stackit-cli/assets/28832884/a0f0b784-ffa1-48f8-b4ca-6eceaa0fc705)